# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/credentialmanagement/tests/browser/browser.toml
# Commit: d07def1365a4
# Full Hash: d07def1365a49dc8ecbf5b6c10fdfbd5ec750932
# Author: pstanciu <pstanciu@mozilla.com>
# Date: 2025-06-11 18:38:51
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing WPT failures @lfedcm-identity.create-store-collect.tentative.sub.https.html
#   
#   This reverts commit 03d4474e360f28be0fb075379f4e4535bd34b252.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 4e25ddaadc86 -r d07def1365a4 toolkit/components/credentialmanagement/tests/browser/browser.toml
--- a/toolkit/components/credentialmanagement/tests/browser/browser.toml	Wed Jun 11 13:42:13 2025 +0000
+++ b/toolkit/components/credentialmanagement/tests/browser/browser.toml	Wed Jun 11 18:04:27 2025 +0300
@@ -2,6 +2,8 @@
 head = "head.js"
 prefs = [
   "dom.security.credentialmanagement.identity.enabled=true",
+  "dom.security.credentialmanagement.identity.heavyweight.enabled=true",
+  "dom.security.credentialmanagement.identity.lightweight.enabled=true",
   "dom.security.credentialmanagement.identity.ignore_well_known=true",
   "privacy.antitracking.enableWebcompat=false",
 ] # disables opener heuristic
@@ -11,4 +13,6 @@
 
 ["browser_account_dialog.js"]
 
+["browser_credential_chooser.js"]
+
 ["browser_provider_dialog.js"]